packages/manager/core/generator/app/templates/src/components/layout-helpers/Listing/DatagridWrapper.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import { OsdsDatagrid } from '@ovhcloud/ods-component-datagrid/react';
+import { OsdsLink } from '@ovhcloud/ods-component-link/react';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import appConfig from '@/{{appName}}.config';
+import ConfigInterface from '@/configInterface';
+import { ReactFormatter } from '@ovh-ux/manager-core-utils';
+
+function LinkService({ cellData, navigate }: any) {
+  return (
+    <OsdsLink
+      color={ODS_THEME_COLOR_INTENT.primary}
+      onClick={() => navigate(`/${cellData}`)}
+    >
+      {cellData}
+    </OsdsLink>
+  );
+}
+
+export default function DatagridWrapper({ data }: any) {
+  const navigate = useNavigate();
+  const myConfig: ConfigInterface = appConfig;
+  const serviceKey = myConfig.listing?.datagrid?.serviceKey;
+  const tmp = Object.assign(data, {});
+  if (tmp && tmp.length === 0) return <div>null</div>;
```

Comment:
Why you render null ? Maybe a message no data ?

Reply:
I delete it, it was a secure, but we dont need it